### PR TITLE
HRIS-223 [FE] Integrate remove schedule functionality

### PR DESCRIPTION
**Pull request by JohnRoyCabezas**
## Issue Link

[Backlog Link](https://framgiaph.backlog.com/board/HRIS?selectedIssueKey=HRIS-223)

## Definition of Done

- [x] Users can delete a schedule
- [x] Users cannot delete a schedule that has employees under it

## Pre-condition

run docker

## Expected Output

Users can delete a schedule
Users cannot delete a schedule that has employees under it

## Screenshots/Recordings

Proof:

[screen-capture.webm](https://user-images.githubusercontent.com/109579325/234832966-f226acf7-a719-4a63-a8d8-63d07d750385.webm)

### CODE REVIEW

File: client/src/components/templates/ScheduleManagementLayout/index.tsx [hunk 1]
```diff
@@ -161,8 +166,24 @@ const ScheduleItem = (item: IGetAllEmployeeSchedule): JSX.Element => {
...
-    onClose()
-    toast.success('Deleted Successfully!')
+    deleteEmployeeScheduleMutation.mutate(
+      {
+        employeeScheduleId: Number(id),
+        userId: user?.userById.id as number
+      },
+      {
+        onSuccess: () => {
+          void queryClient.invalidateQueries()
```
Comment (jsvelte) [inline client/src/components/templates/ScheduleManagementLayout/index.tsx:176]: Please specify the query you need to invalidate.

Comment (jsvelte) [inline client/src/components/templates/ScheduleManagementLayout/index.tsx:176]:
```suggestion
          void queryClient.invalidateQueries(['GET_ALL_EMPLOYEE_SCHEDULE']).then(() => {
              onClose()
              toast.success('Deleted Successfully!')
          })
```

File: client/src/components/templates/ScheduleManagementLayout/index.tsx [hunk 2]
```diff
@@ -161,8 +166,24 @@ const ScheduleItem = (item: IGetAllEmployeeSchedule): JSX.Element => {
...
+        },
+        onSettled: () => {
```
Comment (jsvelte) [inline client/src/components/templates/ScheduleManagementLayout/index.tsx:178]: Please remove onSettled function for the submitting behavior

**Review by jsvelte — CHANGES_REQUESTED** [review]
Please let me know if this will work

**Review by jsvelte — APPROVED** [review]
# Approved 💯

File: client/src/components/templates/ScheduleManagementLayout/index.tsx (hunk 1 above)
Comment (JohnRoyCabezas) [reply client/src/components/templates/ScheduleManagementLayout/index.tsx]: Sir, I added the onClose() function outside the mutation as the modal will not close if ever, we receive an error from the backend.

**Review by abduljalilpalala — APPROVED** [review]
### Approved! ✅

**Pull request merged**